src/backend/base/langflow/components/langwatch/langwatch.py

```
@@ -81,17 +82,9 @@
-    def __init__(self, **data):
-        super().__init__(**data)
-        self.evaluators = self.get_evaluators()
-        self.dynamic_inputs = {}
-        self._code = data.get("_code", "")
-        self.current_evaluator = None
-        if self.evaluators:
-            self.current_evaluator = next(iter(self.evaluators))
-
-    def get_evaluators(self) -> dict[str, Any]:
-        url = f"{os.getenv('LANGWATCH_ENDPOINT', 'https://app.langwatch.ai')}/api/evaluations/list"
+    @lru_cache(maxsize=1)
```

Ruff (0.11.9)

Use of `functools.lru_cache` or `functools.cache` on methods can lead to memory leaks

(B019)